.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+# https://github.com/marketplace/actions/electron-builder-action
+
+on: push
+
+jobs:
+  release:
+    name: Test and Build
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    env:
+      XDG_CACHE_HOME: .cache
+      ELECTRON_CACHE: .cache/electron
+      ELECTRON_BUILDER_CACHE: .cache/electron-builder
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Cache bigger downloads
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: .cache
+          key: ${{ runner.os }}-${{ hashFiles('package.json', 'package-lock.json', 'electron-builder.yml') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('package.json', 'package-lock.json', 'electron-builder.yml') }}
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci --prefer-offline --no-audit --progress=false --cache .cache/npm
+
+      - name: Build
+        run: npm run build
+
+      - name: Test
+        run: npm run test
+
+      - name: Test end-to-end
+        if:  ${{ !startsWith(runner.os, 'Linux') }} # xvfb slow on gh, we run test:e2e on Travis for now
+        run: npm run test:e2e
+
+      - name: Lint
+        run: npm run lint
+
+      # TODO: run electron builder only on release tag? ${{ startsWith(github.ref, 'refs/tags/v') }}
+      - name: Build binaries with electron-builder
+        uses: samuelmeuli/action-electron-builder@92327c67bc45ff7c38bf55d8aa8c4d75b7ea38e7 # v1.6.0 but safer than a tag that can be changed
```

Comment:
This is the version of github action, internally it will use the version of `electron-builder` we have in `package.json`. I don't expect the action to be updated unless there is a breaking-change release of `electron-builder` that we want to switch to. 

ps. I've hardened setup and those third party actions are safelisted in https://github.com/ipfs-shipyard/ipfs-desktop/settings/actions, everything else is blocked. So even if someone sneaks different version in a PR, it won't work.